astronomer-starship/starship/templates/starship/migration.html

```diff
@@ -7,7 +7,7 @@
 <div id="astro-deployment-selector"
      hx-indicator="#astro-deployment-selector-loading"
      hx-get="{{ url_for('AstroMigration.deployment_selector') }}"
-     hx-trigger="load delay:100ms"
+     hx-trigger="load delay:10ms"
```

Comment:
@fritz-astronomer could this cause issues because the tabs are attempting to load too frequently?

Reply:
I don't think so - all the data is behind a cache, and this is just a delay for rendering the component on the page?

Do you mean like a Thundering Herd problem? All of them were on `100ms`, so I think that'd be the same as it ever was.